tests/models/dinov2/test_modeling_dinov2.py

```diff
@@ -168,7 +168,7 @@ def create_and_check_backbone(self, config, pixel_values, labels):
         # verify feature maps
         self.parent.assertEqual(len(result.feature_maps), 1)
         self.parent.assertListEqual(
-            list(result.feature_maps[0].shape), [self.batch_size, self.seq_length, self.hidden_size]
+            list(result.feature_maps[0].shape), [self.batch_size, self.seq_length - 1, self.hidden_size]
```

Comment:
Why did this change? This is possibly breaking for people using the backbone

Reply:
Yes it's a breaking change. However  the backbone is currently not part of the docs, has been added very recently, so I hope you're ok with this change. It now reflects the original implementation, which was not the case before.